_invokers/java.md

```diff
@@ -9,46 +9,180 @@ href: https://github.com/projectriff/java-function-invoker
 
 The Java Invoker hosts riff functions written in Java.
 
-## Usage
+## What It Does
 
-With the `riff` CLI you need to provide an archive location with `--artifact` 
-or `-a` and a handler specification with `--handler`. The archive is a jar
-file, which can be shaded with all required dependencies, or it can be
-a Spring Boot fat jar (with dependencies nested in
-`BOOT-INF/lib`). Simple functions that do not require any dependencies
-work just fine. The simplest form of the handler is a class name that
-can be instantiated (with a default constructor). More complex creation 
-scenarios can be handled by giving the handler in the form `<bean>[&main=<main>]` where
+The "Java Function Invoker" let's you concentrate on writing your business logic as a Java function while the invoker takes care of the rest that is needed to run your functions in a Kubernetes cluster with riff and Knative installed.
+The invoker is a [Spring Boot](https://projects.spring.io/spring-boot) application that will locate your function in the JAR file you provide based on some configuration settings.
```

Comment:
How about saying `riff function create` will compile your function code from source to produce a JAR file which the invoker can load, and it will package the invoker together with the JAR file in a container for deployment in Knative.

Reply:
This part is more explaining the invoker and not so much CLI usage